src/Tools/McpServer/Microsoft.FluentUI.AspNetCore.McpServer.csproj

```diff
@@ -0,0 +1,146 @@
+    
+    <SignAssembly>False</SignAssembly>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>latest</LangVersion>
+
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IsPackable>true</IsPackable>
+
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <DebugType>embedded</DebugType>
+
+    <PackageOutputPath>$(SolutionDir)artifacts</PackageOutputPath>
+
+    <!-- Write the generated files to a specific folder -->
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+
+    <!-- Path to the generated documentation JSON file -->
+    <McpDocumentationJsonPath>$(MSBuildThisFileDirectory)FluentUIComponentsDocumentation.json</McpDocumentationJsonPath>
+  </PropertyGroup>
+
+  <!-- Expose internals to test project -->
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.FluentUI.AspNetCore.McpServer.Tests" />
+  </ItemGroup>
+
+  <!-- Release -->
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
+    <Optimize>True</Optimize>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <!-- NuGet Package Configuration -->
+  <PropertyGroup>
+    <PackageId>Microsoft.FluentUI.AspNetCore.McpServer</PackageId>
+    <Version>5.0.0-preview.6</Version>
+    <PackageIcon>icon.png</PackageIcon>
+    <Authors>
+      Microsoft;dvoituron;aclerbois;vnbaaij
+    </Authors>
+    <Description>MCP (Model Context Protocol) server for Fluent UI Blazor component documentation.
+      Provides AI assistants with access to component details, parameters, events, methods,and enums.</Description>
+    <PackageTags>mcp;model-context-protocol;fluent-ui;blazor;documentation;ai;copilot</PackageTags>
+    <PackageProjectUrl>https://github.com/microsoft/fluentui-blazor</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/microsoft/fluentui-blazor</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+
+
+    <!-- Configure as .NET Tool for NuGet -->
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>fluentui-mcp</ToolCommandName>
+
+    <!-- MCP Server package type for NuGet discovery -->
+    <PackageType>McpServer</PackageType>
+  </PropertyGroup>
+
+  <!-- Include README and MCP server manifest in package -->
+  <ItemGroup>
+    <None Include="README.md" Pack="true" PackagePath="\" />
+    <None Include=".mcp\server.json" Pack="true" PackagePath=".mcp\" />
+    <None Include="..\..\..\icon.png" Pack="true" PackagePath="\" />
+  </ItemGroup>
+
+  <!-- Embed the pre-generated JSON documentation as a resource -->
+
+  <!-- Code Analysis -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
```

Comment:
Can you add this analyzer (already included in Core.csproj)?
You will generate 31 additional errors :-) But it's for a good cause.

```xml
<PackageReference Include="Meziantou.Analyzer">
  <PrivateAssets>all</PrivateAssets>
  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
</PackageReference>
```

Check also the **Message list** containing message like "MA0007"